ui/imports/shared/views/NetworkCardsComponent.qml

```diff
@@ -73,44 +70,34 @@ Item {
-                                   (selectedNetwork && selectedNetwork.chainName === model.chainName) ?
-                                   amountToSend: 0
-                    tertiaryText: qsTr("BALANCE: ") + tokenBalanceOnChain
-                    state: tokenBalanceOnChain === 0 || !hasGas ? "unavailable" : networkCardsComponent.errorMode ? "error" : "default"
-                    cardIcon.source: Style.png(model.iconUrl)
+                    secondaryText: (parseFloat(tokenBalanceOnChain) === 0 && root.amountToSend !== 0) ?
+                                       qsTr("No Balance") : !hasGas ? qsTr("No Gas") : LocaleUtils.numberToLocaleString(fromNetwork.amountToSend)
+                    tertiaryText: qsTr("BALANCE: ") + LocaleUtils.numberToLocaleString(parseFloat(tokenBalanceOnChain))
+                    state: tokenBalanceOnChain === 0 || !hasGas ? "unavailable" : root.errorMode ? "error" : "default"
+                    cardIcon.source: Style.svg(model.iconUrl)
                     disabledText: qsTr("Disabled")
-                    advancedMode: networkCardsComponent.customMode
-                    advancedInputText: (selectedNetwork && selectedNetwork.chainName === model.chainName) ? amountToSend: 0
-                    Component.onCompleted: {
-                        disabled = store.checkIfDisabledByUser(model.chainId)
-                        if(selectedNetwork && selectedNetwork.chainName === model.chainName)
-                            d.selectedFromNetwork = this
-                    }
-                    Connections {
-                        target: networkCardsComponent
-                        onSelectedNetworkChanged: {
-                            if(selectedNetwork.chainName === model.chainName) {
-                                d.selectedFromNetwork = fromNetwork
-                            }
-                        }
-                    }
+                    advancedMode: root.customMode
+                    advancedInputText: LocaleUtils.numberToLocaleString(fromNetwork.amountToSend)
+                    disabled: store.disabledChainIdsFromList.includes(model.chainId)
+                    clickable: root.interactive
                     onClicked: {
-                        store.addRemoveDisabledChain(suggestedRoutes, model.chainId, disabled)
-                        reCalculateSuggestedRoute(store.disabledChainIds)
-                    }
-                    onAdvancedInputTextChanged: {
-                        if(selectedNetwork && selectedNetwork.chainName === model.chainName) {
-                            d.customAmountToSend = isNaN(parseFloat(advancedInputText)) ? 0 : parseFloat(advancedInputText)
-                        }
+                        store.addRemoveDisabledFromChain(model.chainId, disabled)
+                        root.reCalculateSuggestedRoute()
                     }
+//                    onAdvancedInputTextChanged: {
```

Comment:
removed or is it for advance?

Reply:
it will be needed when we work on custom view